src/environment-variables-verification.js

```diff
@@ -0,0 +1,66 @@
+const {
+  VIDEO_PATH,
+  HASH_PATH,
+  TRACE_API_SALT,
+  MILVUS_ADDR,
+  MILVUS_TOKEN,
+  DB_HOST,
+  DB_PORT,
+  DB_USER,
+  DB_PASS,
+  DB_NAME,
+} = process.env;
+
+export function verifyEnvironmentVariables() {
+  if (!VIDEO_PATH || HASH_PATH.length === 0) {
+    console.warn("`VIDEO_PATH` not set.");
+    return false;
+  }
+
+  if (!HASH_PATH || HASH_PATH.length === 0) {
```

Comment:
```suggestion
  if (!HASH_PATH) {
```
No need to check `length === 0` as `""` casts to `false`